.github/update_registry.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+DIRNAME=$(dirname $0)
+REGISTRY=$(realpath $1)
+
+function usage() {
+    echo "Usage: update_registry <registry> <version>"
+    echo "  registry    Path to the vcpkg registry to update."
+    echo "  version     Release version to add to registry."
+}
+
+if [ ! -d ${REGISTRY} ]; then
+    echo "Error: Path to registry does not exist!" >&2
+    usage
+    exit 1
+fi
+
+if [ ! -f ${REGISTRY}/index.yaml ]; then
+    echo "Error: No valid registry found at ${REGISTRY}!" >&2
+    usage
+    exit 1
+fi
+
+if [[ ! "$2" =~ [[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+ ]]; then
+    echo "Error: No valid version identifier '$2'!" >&2
+    usage
+    exit 1
+fi
+
+export CTOOLS_VERSION="$2"
+
+wget https://github.com/Open-CMSIS-Pack/cmsis-toolbox/releases/download/${CTOOLS_VERSION}/cmsis-toolbox-checksums.txt
```

Comment:
For consistency should `curl` be used here since it is used in line 41 (or the other way around)?